packages/react-core/src/components/Alert/AlertIcon.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { css } from '@patternfly/react-styles';
+import PropTypes from 'prop-types';
+import styles from '@patternfly/patternfly-next/components/Alert/styles.css';
+import {
+  CheckCircle,
+  ExclamationCircle,
+  ExclamationTriangle,
+  InfoCircle
+} from '@patternfly/react-icons';
+
+export const incoType = {
+  success: <CheckCircle size="md" />,
+  danger: <ExclamationCircle size="md" />,
+  warning: <ExclamationTriangle size="md" />,
+  info: <InfoCircle size="md" />
+};
+
+const propTypes = {
+  type: PropTypes.oneOf(Object.keys(incoType)).isRequired,
+  isAriaHidden: PropTypes.bool,
+  className: PropTypes.string
+};
+
+const defaultProps = {
+  isAriaHidden: true,
+  className: ''
+};
+
+const AlertIcon = ({ isAriaHidden, type, className, ...props }) => (
+  <div
+    aria-hidden="true"
+    {...props}
+    className={`${css(styles.alertIcon, className)}`}
+  >
+    {incoType[type]}
```

Comment:
this can be simplified by doing:
```jsx
const variantIcons = {
  success: CheckCircle,
  danger: ExclamationCircle,
  warning: ExclamationTriangle, 
  info: InfoCircle
};

const AlertIcon = ({ variant, className, ...props }) => {
  const Icon = variantIcons[variant];
  return (
    <div {...props} className={css(styles.alertIcon, className)}>
      <Icon size="md" />
    </div>
  )